kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/RuntimeDataServiceIntegrationTest.java

```diff
@@ -900,10 +900,17 @@ public void testFindTaskEvents() throws Exception {
 
         try {
 
-            List<TaskSummary> tasks = taskClient.findTasks("yoda", 0, 10);
+            List<TaskSummary> tasks = taskClient.findTasks("yoda", 0, 50);
```

Comment:
Good idea. This return only one task, that we need. I think it should be work OK.
I'll do changes.